### PR TITLE
Jaeger grpc receiver/collector

### PR DESCRIPTION
**Pull request by annanay25**
Initial implementation to receive GRPC spans from Jaeger agents.

### CODE REVIEW

File: receiver/jaeger/trace_receiver.go [hunk 1]
```diff
@@ -47,6 +49,7 @@ import (
 type Configuration struct {
 	CollectorThriftPort int `yaml:"tchannel_port"`
 	CollectorHTTPPort   int `yaml:"collector_http_port"`
+	CollectorGRPCPort   int `yaml:"collector_http_port"`
```
Comment (odeke-em) [inline receiver/jaeger/trace_receiver.go]: Conflict in the YAML tag with `collector_http_port`, please make this `collector_grpc_port`

**Comment by pjanotti** [comment]
Hi @annanay25 thanks for working on this.

Issues #278 and #279 are assigned to @sjkaris. I believe he has #278 mostly done but didn't publish a PR because Jaeger didn't release a version with gRPC support yet. Let's coordinate to see what each of you have ready and possibly reduce the work for both.

File: cmd/occollector/app/builder/builder.go [hunk 2]
```diff
@@ -34,6 +34,8 @@ type JaegerReceiverCfg struct {
 	ThriftTChannelPort int `mapstructure:"jaeger-thrift-tchannel-port"`
 	// ThriftHTTPPort is the port that the relay receives on for jaeger thrift http requests
 	ThriftHTTPPort int `mapstructure:"jaeger-thrift-http-port"`
+	// ThriftGRPCPort is the port that the relay receives for jaeger grpc requests
+	ThriftGRPCPort int `mapstructure:"jaeger-thrift-grpc-port"`
```
Comment (yurishkuro) [inline cmd/occollector/app/builder/builder.go]: jaeger agents use Protobuf w/ gRPC, not Thrift.

Comment (annanay25) [reply cmd/occollector/app/builder/builder.go]: Thanks @yurishkuro I'll keep in mind. :)

**Comment by annanay25** [comment]
Hi, as per the discussion on Gitter, I'm closing this PR. Thanks for the helpful reviews. :)

**Pull request closed without merging**